pkg/attestation/renderer/chainloop/chainloop.go

```diff
@@ -189,6 +193,23 @@ func extractPredicate(statement *intoto.Statement, v *ProvenancePredicateV02) er
 	return nil
 }
 
+func fixPolicyTypeField(statement *intoto.Statement) {
```

Comment:
Needed to prevent marshaling errors with old, wrong attestations with enum (number) `type` field.